scripts/update-data.js

```diff
@@ -0,0 +1,20 @@
+const fs = require("fs");
+const path = require("path");
+const axios = require("axios");
+
+const sourceURL = "https://raw.githubusercontent.com/AngelGarcia13/DominicanWhoCodes/master/DWC.Blazor/wwwroot/data/developers.json";
+axios({
+    method: "GET",
+    url: sourceURL
+}).then(({ data: developers }) => {
+    let developersFileContent = `{ "developers": ${JSON.stringify(developers)} }`;
+    // We're calling JSON.stringify again just to format with 2 space the whole content of the file.
+    developersFileContent = JSON.stringify(JSON.parse(developersFileContent), null, 2);
+
+    fs.writeFile(path.resolve(__dirname, "../developers.json"), developersFileContent, (err => {
+        if (err) throw err;
+        console.log("Developers data updated correctly.");
+    }))
+}).catch(error => {
+    console.error(error);
+})
```

Comment:
I'd like to know, are we using or not `;` constantly in our codebase? Or better put, what are the rules of using it or not?

Here there's no semicolon, although we have at when ending statements.

Reply:
@vyzaldysanchez This should include a `;` and yeah, we need to enforce the usage of `;`  to maintain consistency using eslint